spec/integration/pr/test-pr-1093.spec.js

```diff
@@ -1,33 +0,0 @@
-const proxyquire = require('proxyquire');
```

Comment:
why these files were deleted?

Reply:
this test relies on `proxyquire`, which messes a lot with `verquire`, making some of our tests flaky. I was part of the PR that added this test and can confirm that the thing it tests has become mostly irrelevant with the new control flow. Therefore, it's fine to delete this test and increase the overall test suite stability - but maybe @sohai can confirm?

Reply:
yep, that makes sense to delete those tests now